src/core/ext/debug/debugger_macros.c

```diff
@@ -0,0 +1,71 @@
+/*
+ *
+ * Copyright 2016, Google Inc.
```

Comment:
The year was right when I wrote this code..... (yanked it out of another PR today)